package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "couchbackup",
-  "version": "2.0.0-SNAPSHOT",
+  "version": "2.0.0",
```

Comment:
After the publishing branch merges, this is no longer a suitable way of testing as it will actually cause a release to happen.